mmengine/registry/utils.py

```diff
@@ -90,3 +92,26 @@ def count_registered_modules(save_path: Optional[str] = None,
         dump(scan_data, json_path, indent=2)
         print_log(f'Result has been saved to {json_path}', logger='current')
     return scan_data
+
+
+def init_default_scope(scope: str) -> None:
+    """Initialize the given default scope.
+
+    Args:
+        scope (str): The name of the default scope.
+    """
+    never_created = DefaultScope.get_current_instance(
+    ) is None or not DefaultScope.check_instance_created(scope)
```

Comment:
Is it necessary to check `DefaultScope.get_current_instance()` additionally if `DefaultScope.check_instance_created(scope)` return False?